elasticsearch/sgconfig/action_groups.yml

```diff
@@ -41,6 +41,7 @@ READ:
   - "indices:data/read*"
   - "indices:admin/mappings/fields/get*"
   - "indices:admin/mappings/get"
+  - "indices:data/read/search"
```

Comment:
is there any reason to think someone may also want 'msearch' permissions that would be covered also by `SEARCH` ?